.travis.yml

```diff
@@ -1,20 +1,18 @@
 sudo: false
 language: python
 env:
-- DJANGO="django>1.8,<1.9" DJANGO_SETTINGS_MODULE=representatives_votes.tests.settings
+- DJANGO="django>1.8,<1.9" DJANGO_SETTINGS_MODULE=test.settings
```

Comment:
From the examples I've seen it seems good practice to keep tests in a separate app ; that way, running `django-admin test` on a dependant app won't run our tests.